Gemfile

```diff
@@ -3,7 +3,8 @@ source 'https://rubygems.org'
 gem 'rails', '4.2.5.2'
 gem 'slimmer', '10.0.0'
 
-gem 'govuk_frontend_toolkit', '~> 3.1.0'
+gem 'govuk_elements_rails', '~> 3.0.1'
```

Comment:
We're currently not using GOV.UK Elements on any GOV.UK Publishing frontend applications.

Seems as though the kind of UI you're implementing it makes sense to make use of Elements, just Elements is technically built in a different way to the rest of GOV.UK Publishing frontend.

Maybe let's have a chat through it offline